### PR TITLE
feat: platform use create bookings

### PR DESCRIPTION
**Pull request by ThyMinimalDev**
## What does this PR do?

platform booker create bookings hooks

### CODE REVIEW

File: apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts [hunk 1]
```diff
@@ -56,10 +53,9 @@ export class BookingsController {
   @Post("/reccuring")
   async createReccuringBooking(
     @Req() req: Request & { userId?: number },
-    @Body() _: CreateReccuringBookingInput[],
-    @GetUser() user: User
+    @Body() _: CreateReccuringBookingInput[]
```
Comment (supalarry) [inline apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts:56]: Can we remove @Body() if its not being used "_" here and in other places in controller?

File: apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts [hunk 2]
```diff
@@ -30,17 +27,17 @@ import { ApiResponse } from "@calcom/platform-types";
   path: "ee/bookings",
   version: "2",
 })
-@UseGuards(AccessTokenGuard)
```
Comment (supalarry) [inline apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts:33]: Wonder if public endpoints should be in a separate folder.

**Review by supalarry — APPROVED** [review]
Looks good - just 2 small comments below.

File: apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts (hunk 1 above)
Comment (ThyMinimalDev) [reply apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts:56]: still need to validate the params

**Pull request merged**